private/bufpkg/bufimage/bufimagemodify/php_metadata_namespace.go

```diff
@@ -22,17 +22,24 @@ import (
 	"google.golang.org/protobuf/types/descriptorpb"
 )
 
+// PhpMetadataNamespaceID is the ID of the php_namespace_metadata modifier.
+const PhpMetadataNamespaceID = "PHP_NAMESPACE_METADATA"
```

Comment:
This should be
```suggestion
const PhpMetadataNamespaceID = "PHP_METADATA_NAMESPACE"
```
Right?

Reply:
Actually, turns out our documentation needs to be updated and the ID is indeed `php_metadata_namespace`. Updating :D